llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -478,6 +508,15 @@ std::string saveModuleProperties(module_split::ModuleDesc &MD,
       }
     }
   }
+
+  if (GlobProps.EmitImportedSymbols) {
+    // record imported functions in the property set
+    for (const auto &F : M) {
+      if (isImportedFunction(F))
+        PropSet.add(PropSetRegTy::SYCL_IMPORTED_SYMBOLS, F.getName(), true);
```

Comment:
can we add a comment on what `true` is, like
`/*whatever=*/true`

Reply:
Comment added.